db/schema.rb

```diff
@@ -10,8 +10,50 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema[7.0].define(version: 0) do
+ActiveRecord::Schema[7.0].define(version: 2023_05_02_211542) do
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "comments", force: :cascade do |t|
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.integer "author_id", null: false
+    t.integer "post_id", null: false
+    t.text "text"
+  end
+
+  create_table "likes", force: :cascade do |t|
+    t.bigint "post_id_id", null: false
+    t.bigint "author_id_id", null: false
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["author_id_id"], name: "index_likes_on_author_id_id"
+    t.index ["post_id_id"], name: "index_likes_on_post_id_id"
+  end
+
+  create_table "posts", force: :cascade do |t|
+    t.bigint "author_id_id", null: false
+    t.string "title"
+    t.text "text"
+    t.integer "commentCounter"
+    t.integer "likeCounter"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["author_id_id"], name: "index_posts_on_author_id_id"
+  end
```

Comment:
- [ ] Kindly note that these should be `author_id`, not `author_id_id`.

- [ ] Let's stick to the naming conventions in Ruby. We should use underscore for separating table column name, not camel case. Kindly rename the following table columns `commentCounter` and `likeCounter` to `comments_counter` and `likes_counter` respectively.